content/blog/2020-12-19-LLVM-IR-to-assembly.md

```diff
@@ -0,0 +1,77 @@
++++
+title = "LLVM IR to Assembly"
+[extra]
+bio = """
+   Socrates Wong is a second year PhD student at Cornell who interest lies in Computer Archicture.
+"""
+[[extra.authors]]
+name = "Socrates Wong"
++++
+
+# Modifying LLVM’s IR to assembly on a existing ISA with different number of registers. 
```

Comment:
Probably not necessary to have a second title here? Maybe this should be the overall `title` of the blog post (suitably Title Cased), whereas "LLVM IR to Assembly" is pretty vague.